src/widgetastic/browser.py

```diff
@@ -156,6 +156,13 @@ def url(self, address):
         self.logger.info('Opening URL: %r', address)
         self.selenium.get(address)
 
+    @property
+    def title(self):
+        """Returns current title"""
+        current_title = self.selenium.title
+        self.logger.info('Current title: %r', current_title)
```

Comment:
Could we use `f strings` here?

Reply:
I wouldn't do that. 1. f-strings will limit wt.core to py 3.6+ ; 2. logger module has its own syntax for logging statements.